src/mmw/js/src/analyze/views.js

```diff
@@ -60,9 +61,13 @@ var AnalyzeWindow = Marionette.LayoutView.extend({
     },
 
     showHeaderRegion: function() {
-        this.headerRegion.show(new HeaderView({
-            model: new models.AnalyzeModel({
-                shape: this.model.get('area_of_interest')
+        this.headerRegion.show(new coreViews.AreaOfInterestView({
+            App: App,
+            model: new coreModels.AreaOfInterestModel({
+                shape: this.model.get('area_of_interest'),
+                canGoBack: true,
```

Comment:
Do we want to use snake_case for only those models that are persisted on the server or all Backbone models?